src/runtime/components/data/Table.vue

```diff
@@ -239,7 +239,7 @@ export default defineComponent({
           return
         }
 
-        onSelect(row)
+        selected.value.push(row)
```

Comment:
This completely overwrites the behaviour of a custom 'select handler' that someone might add, right?

Reply:
Yes, you are right. That's my bad. 
It can add in the `onSelect` method, I'll fix it